doc/interop-test-descriptions.md

```diff
@@ -1152,7 +1152,7 @@ Procedures:
       }
     }
     ```
-2. After getting a response, client waits up to 5 seconds to receive a OOB load 
+2. After getting a response, client waits up to 10 seconds to receive a OOB load 
```

Comment:
How do you feel about the Go implementation using a 30 second timeout for the entire process?

Reply:
what does entire process mean?
how about this single verification timeout?

Reply:
okay i see, that sounds fine.

Reply:
This is for all of this procedure (steps 1-5).  The verification of any step can take any or all of that total 30 seconds.

Reply:
Using a total timeout might make the debug harder if test failed, while a separate timeout can pin point the exact failed step. Java does not have other tests using overall timeout as Go does. Since it is clear enough that we are pulling for certain condition to be true, either is acceptable for me.